crates/astria-sequencer/src/bridge/state_ext.rs

```diff
@@ -57,9 +57,12 @@ impl From<&asset::Id> for AssetId {
 struct Fee(u128);
 
 const BRIDGE_ACCOUNT_PREFIX: &str = "bridgeacc";
+const BRIDGE_ACCOUNT_SUDO_PREFIX: &str = "bsudo";
+const BRIDGE_ACCOUNT_WITHDRAWER_PREFIX: &str = "bwithdrawer";
```

Comment:
we use `bridge` as prefix for all other  bridger related things. Should probably be consistent?

Reply:
it just adds more bytes to the db which i want to avoid, maybe we should change all of these